databuilder/extractor/db_api_extractor.py

```diff
@@ -47,7 +47,7 @@ def _execute_query(self) -> Iterable[Any]:
         Use cursor to execute the {sql}
         :return:
         """
-        LOGGER.info('Executing query: \n{}'.format(self.sql))
+        LOGGER.info('Executing query: \n%s', self.sql)
```

Comment:
same here

Reply:
oh wait is this functionality you are leveraging from LOGGER.info?

Reply:
Yep. It's API from `LOGGER.info`